java-checks/src/main/java/org/sonar/java/checks/RedundantTypeCastCheck.java

```diff
@@ -39,14 +46,11 @@
 import org.sonar.plugins.java.api.tree.TypeCastTree;
 import org.sonar.plugins.java.api.tree.VariableTree;
 
-import javax.annotation.CheckForNull;
-
-import java.util.List;
-import java.util.Set;
-
 @Rule(key = "S1905")
 public class RedundantTypeCastCheck extends IssuableSubscriptionVisitor {
 
+  private static final Predicate<JavaSymbol> ABSTRACT_METHOD_PREDICATE = ((Predicate<JavaSymbol> )JavaSymbol::isDefault).negate();
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Remove this unnecessary cast to "Predicate". ([squid:S1905](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS1905))

[See it in SonarQube](https://next.sonarqube.com/sonarqube/project/issues?id=org.sonarsource.java%3Ajava&pullRequest=1969&issues=AWKLTadyAnwv6XEVDnDp&open=AWKLTadyAnwv6XEVDnDp)

[ISSUE_KEY]: AWKLTadyAnwv6XEVDnDp